src/D2L.CodeStyle.Analyzers/Diagnostics.cs

```diff
@@ -543,5 +543,14 @@ public static class Diagnostics {
 			defaultSeverity: DiagnosticSeverity.Error,
 			isEnabledByDefault: true
 		);
+
+		public static readonly DiagnosticDescriptor ConflictingAuditing = new DiagnosticDescriptor(
+			id: "D2L0073",
+			title: "The [Audited] and [Unaudited] attributes are mutually exclusive.",

```

Comment:
Avoid speaking of `[Audited]` and `[Unaudited]` because "those aren't things" (it's `[Statics.Audited]` etc.)

For the title you can make it generic, and then in the `messageFormat` use `[{0}.Audited]` etc. and pass that in from the analyzer.